src/qt/optionsdialog.cpp

```diff
@@ -302,6 +310,15 @@ void OptionsDialog::setMapper()
     mapper->addMapping(ui->proxyIpTor, OptionsModel::ProxyIPTor);
     mapper->addMapping(ui->proxyPortTor, OptionsModel::ProxyPortTor);
 
+    /* Window */
+#ifndef Q_OS_MAC
+    if (QSystemTrayIcon::isSystemTrayAvailable()) {
+        mapper->addMapping(ui->hideTrayIcon, OptionsModel::HideTrayIcon);
+        mapper->addMapping(ui->minimizeToTray, OptionsModel::MinimizeToTray);
+    }
+    mapper->addMapping(ui->minimizeOnClose, OptionsModel::MinimizeOnClose);
+#endif
+
```

Comment:
This adds too much... Maybe missing a prior backport?

https://github.com/bitcoin/bitcoin/pull/14228/files#diff-1d6f5264407b3b2e18be8d9d9653bbf1d67979f4f95dc26575dc23bb8463b4e7L214-R225